_projects/tool_pandt_project.md

```diff
@@ -30,54 +33,72 @@ measurement and analysis tools {% cite GeimerEtAl2010 --file external/tool_pandt
 {% cite FringsEtAl2009 --file external/tool_pandt_project.bib  %}) are two notable examples of 
 tools developed by RIKEN and JSC for this purpose. This project proposes to extend their support for 
 JLESC HPC systems and exploit their capabilities in an integrated work flow.  
-Existing training material will be adapted to collaborators’ large-scale HPC systems, augmented
+Existing training material will be adapted to collaborators large-scale HPC systems, augmented
 with newly prepared material, and refined for better uptake based on participant evaluations
 and feedback. Travel and accommodation expenses of training presenters to participate in
-joint training events (such as VI-HPS Tuning Workshops {% cite ViHPS --file external/tool_pandt_project.bib  %}) will be supported. 
+joint training events (such as VI-HPS Tuning Workshops {% cite VIHPSTWS --file external/tool_pandt_project.bib %}) will be supported. 
 Collaborative work with application developers will assess the effectiveness of the current (and revised) tools,
 and help direct development of new tool capabilities.
 
 ## Results for 2014/2015
 
 JLESC project proposed (and accepted) in April 2015, and planning commenced for initial presentation
-at 3rd JLESCWorkshop (Barcelona) in June 2015. 
+at 3rd JLESC Workshop (Barcelona) in June 2015. 
 RIKEN AICS applied to join VI-HPS with a view to contributing to VI-HPS Tuning Workshops 
-{% cite ViHPS --file external/tool_pandt_project.bib  %} and Tools Guide {% cite ViHPS2015 --file external/tool_pandt_project.bib  %}. 
+{% cite VIHPSTWS --file external/tool_pandt_project.bib %} and Tools Guide {% cite VIHPS2015 --file external/tool_pandt_project.bib %}. 
 XcalableMP training material being translated into English, and Scalasca/Score-P training material being translated
 into Japanese.
 
+## Results for 2015/2016
+
+XcalableMP tutorial held at JSC (2015-12-01).
+Organisation of 20th VI-HPS Tuning Workshop hosted by RIKEN AICS (2016/02/24-26)
+{% cite VIHPSTWS --file external/tool_pandt_project.bib %} 
+covering tools from BSC (Paraver/Extrae/Dimemas) and JSC (Scalasca/Score-P/CUBE)
+on K computer and local Fujitsu FX10 (pi).  
+Performance analysis of RIKEN FIBER mini-app ntchem on pi/K.  
+Performance analysis of ABySS-P {% cite KitayamaEtAl2015 --file external/tool_pandt_project.bib %}
+and NEST neuronal network simulation tool on K computer.
+
```

Comment:
You could use a list here. Start each line with a `*` and keep the blank line no. 53.